### DISCUSSION TITLE
Feature Request: Option to bulk select/edit/delete items within a collection

### DISCUSSION THREAD

**Discussion by matthijs166** [payloadcms/payload#2349] [Feature Requests & Ideas]
As a user of the collection feature in your application, I would like to have the option to select multiple items within a collection and perform bulk actions such as editing or deleting them at once. This would save me time and effort when managing a large number of items within a collection.

The bulk selection feature could be implemented with a checkbox next to each item, which allows the user to select multiple items with a single click. Once selected, the user could then perform actions such as editing the metadata of all selected items simultaneously, or deleting them all at once.

Additionally, it would be useful to have a confirmation prompt before any bulk deletion action is executed, to prevent accidental deletion of important items.

Overall, this feature would greatly improve the user experience of the collection feature and increase productivity for users who frequently manage large collections of items.

**Comment by JarrodMFlesch** [comment]
Hey @matthijs166 this is coming. Sooner than you might think 🙂

**Comment by DanRibbens** [comment]
Updating an old discussion, this was implemented in a prior version and anyone looking for it needs to update Payload to >1.7.0.
